test_scripts/Policies/HMI_PTU/017_Successful_PTU_after_failed_retry_parallel_trigger_propr.lua

```diff
@@ -0,0 +1,114 @@
+---------------------------------------------------------------------------------------------------
```

Comment:
Currently this test case does not pass. 
```
[01:23:39,228] Unsuccessful_PTU_via_a_mobile_device                                                  [FAIL] (60053 ms)
 HMI notification SDL.OnStatusUpdate: Timeout: HMI notification SDL.OnStatusUpdate: Timeout expired
 HMI call BasicCommunication.PolicyUpdate: Timeout: HMI call BasicCommunication.PolicyUpdate: Timeout expired
[01:24:39,282] Successful_PTU_via_HMI                                                                [FAIL] (4313 ms)
 HMI notification SDL.OnStatusUpdate: ValidIf: 
params.status: expected: UP_TO_DATE, actual value: UPDATING
 HMI call BasicCommunication.PolicyUpdate: Times: The most allowed occurences boundary exceed
```
Please explain why this test is necessary and, if so, why it's omitted from the hmi_ptu test set. If it is not necessary please remove this test

Reply:
@ShobhitAd, please see comment above.